tasks/gulp/watch.js

```diff
@@ -5,7 +5,7 @@ const configPaths = require('../../config/paths.json')
 // Watch task ----------------------------
 // When a file is changed, re-run the build task.
 // ---------------------------------------
-gulp.task('watch', () => {
```

Comment:
I don't believe this change is necessary as it was a problem with my computer, sorry!

Reply:
If an error does happen in `watch` in future you'll want the promise to reject and be fed back to gulp so it can handle it correctly. Sure you don't want to leave it in?

Reply:
Sounds useful to me so can leave it in sure.